website/content/specification/syntax/constraints.md

```diff
@@ -179,6 +179,16 @@ A constraint may have an OPTIONAL [`@level`](#level) attribute and/or an OPTIONA
 
 If defined, the `<message>` value MUST be a [Metaschema string value](/specification/datatypes#string). It MAY contain a Metapath expression templates that starts with `{`, contains a Metapath expression, and ends with `}`.  When evaluating a template Metapath expression, the context of the Metapath [evaluation focus](#constraint-processing) MUST be the failing value node.
 
+##`matches`
+
+The `matches` constraint is a type of Metaschema constraint that restricts field or flag value(s) based on a component matching the target Metapath expression. Each one of these are discussed below.
+
+A match can be made by 2 different ways based on `@datatypes` and based on `@regex`. 
+
+The `@target` attribute of an `matches` constraint specifies the node in a component definition whose value is equal. The `matches` constraint can define a [`target`](#target) with a Metapath expression. The targeted node MUST also match on `@datatype` to return a TRUE. 
+
+The `@target` attribute of a `matches` constraint can be matched based on its value and a `@regex` that the node matches on.
```

Comment:
Blocking: it is good we added a statement about `@regex`, but we need much more detail on minimally required description of how it can be used and brief summary explanation of the syntax, [linking to here](https://pages.nist.gov/metaschema/specification/datatypes/#regular-expressions-in-schemas).

(**NOTE**: It is important we circle back on #325 and update the link to the datatypes anchor to more expansive information once created on a new detailed page per this cross-related issue. Writing this to create a backlink in 325 to integrate and re-review cohesion with this info later.)

Reply:
Ditto here with https://github.com/usnistgov/metaschema/pull/525/commits/34ce6424a492f28c77334e14b93b53b96620d160.